src/transformers/models/videomae/feature_extraction_videomae.py

```diff
@@ -159,8 +162,20 @@ def __call__(
             videos = [self.resize_video(video, size=self.size, resample=self.resample) for video in videos]
         if self.do_center_crop and self.size is not None:
             videos = [self.crop_video(video, size=self.size) for video in videos]
+
+        # if do_normalize=False, the casting to a numpy array won't happen, so we need to do it here
```

Comment:
I would maybe just write in the comment here:
# cast to numpy array

and then **not** call `self.to_numpy_array` in `normalize_video` (see comments above) think this could make this code easier to read/understand